src/document/DocumentCommandHandlers.js

```diff
@@ -159,8 +168,11 @@ define(function (require, exports, module) {
             windowTitle         = brackets.config.app_title,
             currentlyViewedFile = MainViewManager.getCurrentlyViewedFile(MainViewManager.ACTIVE_PANE),
             currentlyViewedPath = currentlyViewedFile.fullPath,
-            readOnlyString      = currentlyViewedFile.readOnly ? "[Read Only] - " : "";
-
+            readOnlyString      = currentlyViewedFile.readOnly ? "[Read Only] - " : "",
+            filepathDisplayPref = PreferencesManager.get('showFullFilePath');
+            // Toggles between display of full or truncated filepath in top bar based on setting preference
+            var pathToFile = filepathDisplayPref ? _currentTitlePath : currentlyViewedPath;
```

Comment:
- On setting filepathDisplayPref  to true, pathToFile should be set to currentlyViewedPath not _currentTitlePath.
- You don't have to use an extra var keyword.
- Can you change the name of variable 'pathToFile' to something related to Title as was for previous name '_currentTitlePath'